infra/resources/_modules/apim/send.tf

```
@@ -0,0 +1,60 @@
+resource "azurerm_api_management_product" "apim_itn_product_send_aar" {
+  product_id   = "io-send-aar-api"
+  display_name = "IO SEND AAR API"
+  description  = "Product for IO SEND AAR API"
+
+  api_management_name = data.azurerm_api_management.apim_itn_api.name
+  resource_group_name = data.azurerm_api_management.apim_itn_api.resource_group_name
+
+  approval_required = false
+  published         = true
+}
+
+resource "azurerm_api_management_product_policy" "apim_itn_product_send_policy" {
+  product_id          = azurerm_api_management_product.apim_itn_product_send_aar.product_id
+  api_management_name = data.azurerm_api_management.apim_itn_api.name
+  resource_group_name = data.azurerm_api_management.apim_itn_api.resource_group_name
+
+  xml_content = file("../_modules/apim/product/policy.xml")
+}
+
+data "http" "send_aar_openapi" {
+  url = "https://raw.githubusercontent.com/pagopa/io-messages/d80ef63a1ddf88eec5753ad67007cd365496b2b5/apps/send-func/openapi/aar-notification.yaml"
+}
+
+resource "azurerm_api_management_api" "send_api_v1" {
+  name                = format("%s-%s-send-aar-api-01", local.product, var.legacy_location_short)
+  api_management_name = data.azurerm_api_management.apim_itn_api.name
+  resource_group_name = data.azurerm_api_management.apim_itn_api.resource_group_name
+  revision            = "1"
+
+  description  = "IO Send AAR - API"
+  display_name = "IO Send AAR - API"
+
+  path      = "api/v1/send"
+  protocols = ["https"]
+
+  subscription_required = true
+  service_url           = null
+
+  import {
+    content_format = "openapi"
+    content_value  = data.http.send_aar_openapi.response_body
+  }
```

You can directly pass the opanapi url without using a data

```suggestion
  import {
    content_format = "openapi-link"
    content_value  = "https://raw.githubusercontent.com/pagopa/io-messages/d80ef63a1ddf88eec5753ad67007cd365496b2b5/apps/send-func/openapi/aar-notification.yaml"
  }
```

As we did [here](https://github.com/pagopa/io-messages/blob/feat/send-aar-qrcode-and-notification-api/infra/resources/_modules/apim/services.tf#L60-L60) for example

---

a2df4980185e3570d6b27eaaf960cc1ee3862989